sdccc/src/main/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditions.java

```diff
@@ -398,6 +408,29 @@ static Optional<String> associateNewLocationForHandle(
             return stateHandle;
         }
 
+        private static void waitForEpisodicContextReportToModifyStateHandle(
```

Comment:
I would consider moving the wait logic to waitForStateUpdate

Reply:
Done.